packages/core/src/shared/utilities/textUtilities.ts

```diff
@@ -119,11 +118,8 @@ export function getStringHash(text: string | Buffer): string {
     return hash.digest('hex')
 }
 
-/**
- * Temporary util while Cloud9 does not have codicon support
- */
 export function addCodiconToString(codiconName: string, text: string): string {
```

Comment:
might be able to drop this function, maybe a comment that mentions "cloud9" should stay on it